runtime/io/common/src/aws/smithy/kotlin/runtime/io/SdkByteChannel.kt

```diff
@@ -6,11 +6,12 @@
 package aws.smithy.kotlin.runtime.io
 
 import aws.smithy.kotlin.runtime.util.InternalApi
-import io.ktor.utils.io.ByteChannel as KtorByteChannel
-import io.ktor.utils.io.ByteReadChannel as KtorByteReadChannel
+
+public const val DEFAULT_BYTE_CHANNEL_MAX_BUFFER_SIZE: Int = 8192 * 2
```

Comment:
**Style**: Why write this as `8192 * 2` rather than `16384` or even `16 * 1024`?

Reply:
Do you have a preference lol? The reason it's written this way was in my head I was thinking in terms of okio segment lengths (8k), which is an internal detail but should result in a decent default).

Reply:
Sure that makes sense as a good default and we can even leave it as `8192 * 2`. Let's just add a comment about where 8k is coming from so that it's clearer for future readers.